libraries/AP_Frsky_Telem/AP_Frsky_SPort.cpp

```diff
@@ -66,8 +66,12 @@ void AP_Frsky_SPort::send(void)
             case SENSOR_ID_FAS: // Sensor ID  2
                 switch (_SPort.fas_call) {
                 case 0:
-                    send_sport_frame(SPORT_DATA_FRAME, FUEL_ID, (uint16_t)roundf(_battery.capacity_remaining_pct())); // send battery remaining as %
-                    break;
+                    {
+                        int8_t percentage = 0;
+                        IGNORE_RETURN(_battery.capacity_remaining_pct(percentage));
+                        send_sport_frame(SPORT_DATA_FRAME, FUEL_ID, (uint16_t)roundf(percentage)); // send battery remaining
```

Comment:
Funky stuff happens here if we ever do populate `percentage` with a negative value within `capacity_remaining_pct`

Possibly other places I haven't spotted....

Reply:
Isn't the `constrain_float()` [here](https://github.com/ArduPilot/ardupilot/pull/13712/files#diff-54081b19c44a7421856175f92bb9a689647a3032b0a843326be4d166adc93ec3R41) enough to guarantee it is never negative?